Makefile

```diff
@@ -740,12 +743,23 @@ $(KUBECTL): ## Build kubectl from tools folder.
 	ln -sf $(KUBECTL) $(TOOLS_BIN_DIR)/$(KUBECTL_BIN)
 	chmod +x $(KUBECTL) $(TOOLS_BIN_DIR)/$(KUBECTL_BIN)
 
+$(HELM): ## Put helm into tools folder.
+	mkdir -p $(TOOLS_BIN_DIR)
+	rm -f "$(TOOLS_BIN_DIR)/$(HELM_BIN)*"
+	curl -fsSL -o $(TOOLS_BIN_DIR)/get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
```

Comment:
Do we want to pin a version to eliminate chance that something in helm breaks tests?

Reply:
The trade-off is:

1. Manually update helm versions regularly to ensure you have a patched version
2. Permissively get the latest

Based on the maturity of the helm project, I prefer option 2. Easy to do option 1 if that's what folks want.

Reply:
This comment made me think of something I missed before: the way the Makefile works, because the binary is unversioned, it will install it once and then never install it again if the file "hack/tools/bin/helm" is already present. This means unless devs run `make clean` they won't get the latest version of Helm and their local hack/tools version will get stale. That's the reason why we use a versioned binary for the other tools, with a symlink to the simple binary name so it can be referenced in scripts without the version needing updating whenever we update the version.

For that reason, I'm voting we do 1) and pin to a version so that we ensure the local version folks use is always up to date with our CI version, and follow the same pattern as other hack/took binaries.

Reply:
Approach #1 implemented